packages/fhir-converter/src/templates/cda/ValueSet/SystemReference.hbs

```diff
@@ -17,12 +23,44 @@
         "http://hl7.org/fhir/sid/us-medicare"  
     {{else if (eq code '2.16.840.1.113883.4.927')}}
         "http://hl7.org/fhir/sid/us-mbi"  
-    {{else if (eq code '2.16.840.1.113883.6.59')}}
```

Comment:
we will still need to handle this one? https://terminology.hl7.org/2.1.0/CodeSystem-CVX.html

Reply:
the code was updated. The version you posted there is from hl7v2 of 
http://hl7.org/fhir/sid/cvx and the version I changed it to is hl7v5. We could support both, but if we did that might become a rabbit hole? Will add the old one though but just something to think about

Reply:
yeah, we should probably support both - it was updated, but older records will use older codes

in CDA it should be hl7v3 regardless

Reply:
kk I added